docs/advanced/node-local-storage.md

```diff
@@ -0,0 +1,356 @@
+# Using Node Local Storage on Kubernetes
+
+This guide shows different ways to use node local storage on Lokomotive, to achieve near
+bare-metal storage performance inside Kubernetes pods.
+
+Before you continue, make sure to read generalities about these concepts:
+ * [Volumes](https://kubernetes.io/docs/concepts/storage/volumes/)
+ * [Persistent Volumes and PVC](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+ * [StorageClasses](https://kubernetes.io/docs/concepts/storage/storage-classes/)
+
+More details about specific volumes types used and how to use them on Lokomotive
+will follow, so there is no need to look into volume specifics on the first
+read.
+
+**NOTE**: This has been tested on Packet provider only, some Lokomotive changes may need to be
+ported to other providers for this to work.
+
+## Local Persistent Volumes in Filesystem Mode
+
+This way of using node local storage works by exposing a filesystem mounted in
+the host as a Persistent Volume to pods.
+
+One Persistent Volume can be bound to only one PVC, so only one pod will be able
```

Comment:
Too many `one`s here it seems. How about:
```suggestion
Single Persistent Volume can be bound to only one PVC, so only one pod will be able
```

Reply:
s/Single/A single/?